instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py

```diff
@@ -402,6 +532,16 @@ def instrumented_urlopen(wrapped, instance, args, kwargs):
                 sem_conv_opt_in_mode,
             )
 
+            if span.is_recording():
+                response_headers_to_set = get_custom_header_attributes(
+                    response.headers,
+                    captured_response_headers,
+                    sensitive_headers,
+                    normalise_response_header_name,
+                )
+                for header, value in response_headers_to_set.items():
+                    span.set_attribute(header, value)
```

Comment:
nit: any reason not to just do this
```suggestion
                span.set_attributes(get_custom_header_attributes(
                    response.headers,
                    captured_response_headers,
                    sensitive_headers,
                    normalise_response_header_name,
                ))
```